_articles/dual-booting.md

```diff
@@ -13,50 +13,13 @@ section: pop-ubuntu
 
 ---
 
-To dual boot Pop!\_OS alongside another OS, use the "Customize Partitions…" option when selecting a drive.
+### If Pop!\_OS is installed first
 
-![Customize Partitions](/images/dual-booting/customize.png)
-
-## Create a partition
-
-If you already have an available partition that you want to erase and replace with Pop!\_OS, skip to the next section.
-
-If you don't have an available partition, you can create one using the "Modify Partitions…" button to open GParted.
-
-![Modify Partitions](/images/dual-booting/modify.png)
-
-First, make sure you Select the correct drive in the top-right of GParted.
+To dual boot Pop!\_OS alongside another OS install Pop!\_OS first. Then once booted into Pop!\_OS and use GParted to resize the root partition (the largest partition) to make room for Windows 10 (35GB is the minimum required).  
```

Comment:
Proofreading note: 

"Then, once booted into Pop!\_OS, use Gparted..." [remove "and" so it doesn't read "then once...and use," but "once booted...use" instead.]